pegasus/test/test_geocoder.rb

```diff
@@ -9,21 +9,19 @@ class GeocoderTest < Minitest::Test
   def test_finding_potential_addresses
     Geocoder.configure lookup: :mapbox, api_key: nil
     assert_nil(Geocoder.find_potential_street_address('this is just some text'))
-    assert(Geocoder.find_potential_street_address('1 Embarcadero Blvd SF CA'))
     assert_equal('1 Embarcadero Blvd SF CA', Geocoder.find_potential_street_address('1 Embarcadero Blvd SF CA'))
     assert_equal('123 Post Road Westport CT', Geocoder.find_potential_street_address('Hi I live at 123 Post Road Westport CT'))
     assert_equal('123, Post Road, Westport, CT', Geocoder.find_potential_street_address('Hi I live at 123, Post Road, Westport, CT'))
-    assert_equal('123, Post Road, Westport, CT and other stuff', Geocoder.find_potential_street_address('Hi I live at 123, Post Road, Westport, CT and other stuff'))
+    assert_nil(Geocoder.find_potential_street_address('Hi I live at 123, Post Road, Westport, CT and other stuff'))
```

Comment:
Note that the expected result of this test changed.  With the new threshold, we are no longer able to detect that this string has an address.

Reply:
[Non-blocking comment] Huh. That seems like it should detect an address. Would dropping the threshold lower make this detectable? (basically I'm just wondering why it's ok for this one to return nil when the others detect an address)

Reply:
Good question.  There's more explanation in this [jira comment](https://codedotorg.atlassian.net/browse/LP-1616?focusedCommentId=17198) but basically there's no threshold value that will work for all cases.  As a short-term fix, we've chosen a threshold that allows reasonable names to be accepted and then we'll follow up on a longer-term fix to close the gaps that introduces.